test/Sema/package_symbol_with_default_arg.swift

```diff
@@ -0,0 +1,36 @@
+// RUN: %empty-directory(%t)
+// RUN: split-file %s %t
+
+// RUN: %target-build-swift-dylib(%t/%target-library-name(Utils)) %t/Utils.swift -module-name Utils -package-name mypkg -emit-module -emit-module-path %t/Utils.swiftmodule -emit-tbd -emit-tbd-path %t/libUtils.tbd -emit-sil -o %t/utils.sil
+// RUN: %FileCheck %s --check-prefix CHECK-TBD < %t/libUtils.tbd
+// CHECK-TBD-NOT: _$s5Utils6pubBar3argS2i_tFfA_
+// CHECK-TBD: _$s5Utils6pkgBar3argS2i_tF
+// CHECK-TBD: _$s5Utils6pkgBar3argS2i_tFfA_
+// CHECK-TBD: _$s5Utils6pkgFoo3argS2i_tF
+// CHECK-TBD: _$s5Utils6pubBar3argS2i_tF
+// CHECK-TBD: _$s5Utils6pubFoo3argS2i_tF
+
+// RUN: %target-build-swift -I %t -L %t -lUtils %s -o %t/main %target-rpath(%t) -package-name mypkg
+
+// RUN: %target-run %t/main %t/%target-library-name(Utils) > %t/run-result.output
```

Comment:
Seems like this test may need a `REQUIRES: executable_test`. I would also organize it under `test/Interpreter` rather than `test/Sema`.

Reply:
will do

Reply:
does the test really need to be executable? I think test/TBD/ is the best place for it and it should not be executable, just check that the symbol appears in the TBD.

Reply:
ou don't even need to write any `CHECK:` lines in a TBD test for this, the `-validate-tbd-against-ir` infrastructure does all the validation automatically by matching up the tbd contents and LLVM IR.

Reply:
you're right we don't need an executable; removed.